packages/bundler-plugin-core/src/types.ts

```diff
@@ -47,7 +48,7 @@ export interface Output {
 
 export interface BundleAnalysisUploadPluginArgs {
   output: Output;
-  userOptions: Options;
+  options: NormalizedOptions;
 }
 
 export interface Options {
```

Comment:
should we add a note to `bundleName` doc string about what valid bundle names are?